app/controllers/application_controller.rb

```diff
@@ -1,2 +1,11 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  include SessionsHelper
+
+  def authenticate_user
+    if current_user == nil
```

Comment:
`.nil?`を使った方がRubyでよく記載するやり方に近いですね。
```suggestion
    if current_user.nil?
```